CHANGELOG.md

```diff
@@ -3,6 +3,112 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.12.0
+
+_Aug 1, 2024_
+
+### 💵 Our commercial offering is evolving!
+
+The pro plan is receiving two new packages:
+
+- `@mui/x-tree-view-pro` (available today!)
+- `@mui/x-charts-pro` (available in the coming weeks)
```

Comment:
Can you also update the [package list here](https://mui.com/x/introduction/licensing/#pro-plan)?

Reply:
Added the tree-view-pro there